distributions/Lakka/options

```diff
@@ -319,7 +319,8 @@
                  "
 
 # disable some cores
-    if [ "${PROJECT}" = "RPi" -a "${DEVICE}" = "RPi" -o "${DEVICE}" = "GPICase" ]; then
+  if [ "${PROJECT}" = "RPi" ]; then
+    if [ "${DEVICE}" = "RPi" -o "${DEVICE}" = "GPICase" ]; then
```

Comment:
Why you are touching these? They have nothing to do with your commit message

Reply:
There are 2 reasons.
(1) I know you changed it in last commit, but it is may not good compared to more old one.
I mean " if ( A and B or C ) " is may not judged correctly.
- ( A anb B ) or ( A and C ) ?
- ( A and B ) or ( C ) ?

(2) Please refer to all of distributions/Lakka/options file Line 372/373 and 374/375.
372/373 elif [ "${DEVICE}" = "RPi2" ]; then
374/375 elif [ "${DEVICE}" = "Pi02GPi" ]; then
These "elif" have not <"${PROJECT}" = "RPi" -a> in currently.

Therefore I restored it.

Reply:
A=PROJECT  B or C is DEVICE so if a and if b or c and is perfectly fine as it is

Reply:
I've got it